lib/CtrlNest/Helper/AccessCode.pm

```diff
@@ -0,0 +1,223 @@
+
+# @brief Validates the access code expiration duration by checking it against
+#   defined constraints. Also, if the parameter is valid, it will be converted
+#   from a human-readable format to seconds (integer).
+#
+# @param $access_code_expires_in - The human-readable expiration duration
+#   (e.g., '30m', '7d', 'never', etc).
+#
+# @return
+#   - The corresponding ACCESS_CODE_EXPIRES_IN_X_SECONDS constant, if valid.
+#   - 0 (INVALID) if the expiration duration is invalid.
+#
+sub validate_ac_expires_in {
+  my ($access_code_expires_in) = @_;
+
+  # Must be defined
+  return INVALID unless defined $access_code_expires_in;
+
+  # Expires in 10m
+  return ACCESS_CODE_EXPIRES_IN_10_MIN_SECONDS
+    if ($access_code_expires_in eq ACCESS_CODE_EXPIRES_IN_10_MIN);
+
+  # Expires in 30m
+  return ACCESS_CODE_EXPIRES_IN_30_MIN_SECONDS
+    if ($access_code_expires_in eq ACCESS_CODE_EXPIRES_IN_30_MIN);
+
+  # Expires in 60m
+  return ACCESS_CODE_EXPIRES_IN_60_MIN_SECONDS
+    if ($access_code_expires_in eq ACCESS_CODE_EXPIRES_IN_60_MIN);
+
+  # Expires in 1d
+  return ACCESS_CODE_EXPIRES_IN_1_DAY_SECONDS
+    if ($access_code_expires_in eq ACCESS_CODE_EXPIRES_IN_1_DAY);
+
+  # Expires in 7d
+  return ACCESS_CODE_EXPIRES_IN_7_DAY_SECONDS
+    if ($access_code_expires_in eq ACCESS_CODE_EXPIRES_IN_7_DAY);
+
+  # Expires in 30d
+  return ACCESS_CODE_EXPIRES_IN_30_DAY_SECONDS
+    if ($access_code_expires_in eq ACCESS_CODE_EXPIRES_IN_30_DAY);
+
+  # Expires in never
+  return ACCESS_CODE_EXPIRES_IN_NEVER_SECONDS
+    if ($access_code_expires_in eq ACCESS_CODE_EXPIRES_IN_NEVER);
+
+  return INVALID;    # Fail
+}
+
+################################################################################
+
+# @brief Validates an access code reusability by checking it against
+#   defined constraints. Also, if the parameter is valid, it will be converted
+#   from a human-readable format to bit.
+#
+# @param $access_code_is_reusable - The human-readable reusability value.
+#
+# @return
+#   - 1 if the reusability is valid.
+#   - 0 if the reusability is invalid.
+#
+sub validate_ac_is_reusable {
+  my ($access_code_is_reusable) = @_;
+
+  # Must be defined
+  return INVALID_CHECKBOX unless defined $access_code_is_reusable;
+
+  # Must be one of the defined constant value
+  return INVALID_CHECKBOX
+    unless ($access_code_is_reusable eq CHECKBOX_CHECKED)
+    or ($access_code_is_reusable eq CHECKBOX_UNCHECKED);
+
+  # Check if the data was sent correctly
+  # but the checkbox was not checked
+  return INVALID unless ($access_code_is_reusable eq CHECKBOX_CHECKED);
+
+  return SUCCESS;    # Success
+}
+
+################################################################################
+
+# @brief Validates an access code title by checking it against defined
+#   constraints.
+#
+# @param $title - The access code title to be validated.
+#
+# @return
+#   - 1 if the title is valid.
+#   - 0 if the title is invalid.
+#
+sub validate_ac_title {
```

Comment:
Should also return the validate `title` string. This ensures consistence across the project.